src/main/java/org/osc/manager/ism/entities/DeviceEntity.java

```diff
@@ -39,6 +42,10 @@
     private String name;
     @OneToMany(mappedBy = "device", fetch = FetchType.EAGER)
     private List<DeviceMemberEntity> deviceMembers = new ArrayList<DeviceMemberEntity>();
+    @OneToMany(mappedBy = "sgDevice", fetch = FetchType.EAGER)
```

Comment:
I think `sgDevice` -> `device` is fine since the field `device` is already in the SGEntity no need to prefix it. We dont prefix the other *Entity fields

Reply:
ok